src/components/ui/autocomplete/autocomplete.spec.tsx

```diff
@@ -48,41 +47,52 @@ jest.mock('react-native', () => {
 
 describe('@autocomplete-item: component checks', () => {
 
-  const TestAutocompleteItem = (props?: AutocompleteItemProps) => (
+  const TestAutocompleteItem = (props?: AutocompleteItemProps): JSX.Element => (
```

Comment:
all specs return `JSX.Element`, but in the rest code we use `React.ReactElement`. does it make sense to make return type consistent?